dir.props

```diff
@@ -42,15 +42,23 @@
     <DebugSymbols Condition="'$(DebugSymbols)' == ''">true</DebugSymbols>
     <Optimize Condition="'$(Optimize)' == ''">false</Optimize>
     <DebugType Condition="'$(DebugType)' == ''">full</DebugType>
+    <MsbuildDebugger>true</MsbuildDebugger>
     <DefineConstants>$(DefineConstants);DEBUG;TRACE;STANDALONEBUILD</DefineConstants>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)' == 'Release'">
     <DebugSymbols Condition="'$(DebugSymbols)' == ''">true</DebugSymbols>
     <Optimize Condition="'$(Optimize)' == ''">true</Optimize>
     <DebugType Condition="'$(DebugType)' == ''">pdbonly</DebugType>
+    <MsbuildDebugger>true</MsbuildDebugger>
     <DefineConstants>$(DefineConstants);TRACE;STANDALONEBUILD</DefineConstants>
   </PropertyGroup>
 
+  <!-- Change define constants as needed -->
+  <PropertyGroup>
+    <DefineConstantsMsbuildDebugger Condition="'$(MsbuildDebugger)' == 'true'">MSBUILD_DEBUGGER</DefineConstantsMsbuildDebugger>
```

Comment:
Could you rename to follow the convention started in xplat dirs.props? Assume this will need to be merged there right away. Also rather than debugger as the feature, we might want to go with the reason it needs to be disabled. So Reflection Emit?